app/assets/javascripts/crosswalks.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```

Comment:
@ayaleloehr I was going to remove all the *.js, *.scss , helper files, etc in one fell swoop... But I can do that now...